src/main/java/world/bentobox/bentobox/BentoBox.java

```diff
@@ -201,10 +201,7 @@ public void onDisable() {
         if (islandsManager != null) {
             islandsManager.shutdown();
         }
-        // Save settings
-        if (settings != null) {
-            new Config<>(this, Settings.class).saveConfigObject(settings);
-        }
+        // Save settings - not required
```

Comment:
This has nothing to do with the paste - I just realized it's not needed (yet)

Reply:
What was it expected to do?

Reply:
I don’t know. I think it was left over from when BSB was part of the plugin. In theory, if any of the settings were changed in-game then we’d need it.

Reply:
This needs to go back in like I said. Oops. It's there to save new settings to config.